config.yml

```diff
@@ -0,0 +1,79 @@
+# Acoustic Multiclass Training config file
+
+# Required path related config
+dataframe_csv:
+data_path:
+
+# Dataframe column names
+offset_col: "OFFSET"
+duration_col: "DURATION"
+file_name_col: "FILE NAME"
+manual_id_col: "MANUAL ID"
```

Comment:
Are we standardizing species to be "MANUAL ID"? Is this usually scientific or eBird? This matters for postprocessing from chunking_methods

Reply:
I'm fine with manual id